docroot/sites/all/modules/dev/ws_pm_email_notify/ws_pm_email_notify.module~

```diff
@@ -0,0 +1,607 @@
+<?php
```

Comment:
@yonas To reduce confusion, we should move this extraneous file from the modules directory, since there is already a `ws_pm_email_notify.module` which Drupal utilizes. Someone would need to diff the two files to see what changes if any need to be kept from the `~` file.